src/views/Range/RangeChart.tsx

```diff
@@ -128,19 +131,19 @@ const RangeChart = (props: {
           <>
             <DataRow
               title="Upper Wall"
-              balance={formatCurrency(parseBigNumber(rangeData.wall.high.price, 18), 2, reserveSymbol)}
+              balance={formatCurrency(parseBigNumber(rangeData.high.wall.price, 18), 2, reserveSymbol)}
```

Comment:
Why not reuse defined variables above, i.e. formattedWallHigh

Reply:
formattedWallHigh is a string, but formatCurrency requires a number. 

if we convert string to number and it has decimals, will create potential issues w/ conversion. 

we could create a helper that is just for parseBigNumber(rangeData.high.wall.price, 18) , and then trim or formatCurrency based on that.